torchvision/datasets/voc.py

```diff
@@ -3,6 +3,7 @@
 import collections
 from .vision import VisionDataset
 import xml.etree.ElementTree as ET
+import defusedxml.ElementTree as DET
```

Comment:
If we were to merge this, I wouldn't make `defusedxml` a hard dependency of torchvision, and would instead do something like
```python
try:
    import defusedxml.ElementTree as ET
except ImportError:
    import xml.etree.ElementTree as ET
```
this way we don't force another dependency on users who don't have it

Reply:
Thank you @fmassa . I have added your suggested change. However I had to add it in `def __getitem__` as `defusedxml.ElementTree.Element` does not exist, which is required in `def parse_voc_xml`.